src/Sdk/WebApi/WebApi/ResultsHttpClient.cs

```diff
@@ -81,6 +81,32 @@ private async Task<GetSignedStepLogsURLResponse> GetStepLogUploadUrlAsync(string
             return await GetResultsSignedURLResponse<GetSignedStepLogsURLRequest, GetSignedStepLogsURLResponse>(getStepLogsSignedBlobURLEndpoint, cancellationToken, request);
         }
 
+        private async Task<GetSignedDiagnosticLogsURLResponse> GetSignedDiagnosticLogsURL(string planId, string jobId, CancellationToken cancellationToken)
```

Comment:
I think this and the method below it are doing the same thing. This one can be deleted